addons/PseudoCode_Console/src/presenter.js

```diff
@@ -104,6 +104,8 @@ function generateJumpInstruction(code, toLabel) {
 /***/ }),
 /* 2 */,
 /* 3 */
+/* 3 */,
+/* 4 */
```

Comment:
nie commituj zmian w pseudocode console, wywal to

Reply:
Aj, sorki, usuwałem poprzednie zmiany ręcznie patrząc na to co mi pokazywał github, musiałem nie zauważyć tych dwóch linijek. Zaraz to poprawię